### PR TITLE
Xcorner 18 contact us

### PR DESCRIPTION
**Pull request by mgbelegu**
#### Changes:

- Styling of the contact page in BigCommerce

### CODE REVIEW

File: assets/scss/pages/contact-page.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
+.c-contact-page__section {
```
Comment (marcinkrzeminski) [inline assets/scss/pages/contact-page.scss]: @mgbelegu please use `p-` prefix for pages.

File: assets/scss/pages/contact-page.scss [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+.c-contact-page__title {
+    margin: 0;
+    color: $color-black;
+    font-family: $font-semi-bold;
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.25;
+}
```
Comment (marcinkrzeminski) [inline assets/scss/pages/contact-page.scss]: @mgbelegu can't we use the `c-common__title` here from the other pull request?

File: assets/scss/pages/contact-page.scss [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+        @include bp(medium) {
+            width: fit-content;
+        }
+    }
+
+    br {
+        display: none;
+    }
+}
```
Comment (marcinkrzeminski) [inline assets/scss/pages/contact-page.scss]: @mgbelegu move this to a separate component.

File: assets/scss/pages/contact-page.scss [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+    @include bp(medium) {
+        max-width: 48.875rem;
+    }
+
+    p {
+        &:last-of-type {
+            margin-bottom: 0;
+        }
+    }
```
Comment (marcinkrzeminski) [inline assets/scss/pages/contact-page.scss]: @mgbelegu remove this one please, since it's going to be handled by the `c-wysiwyg` component.

**Review by marcinkrzeminski — CHANGES_REQUESTED** [review]

**Comment by mgbelegu** [comment]
@marcinkrzeminski After rebase most things have been fixed. Can you take another look please?

**Pull request closed without merging**